pkg/artifactory/util_test.go

```diff
@@ -245,3 +247,30 @@ var deleteProxy = func(t *testing.T, proxyKey string) {
 		return []byte(`proxies: ~`)
 	})
 }
+
+func addTestCertificate(t *testing.T, certificateAlias string) {
+	restyClient := getTestResty(t)
+
+	certFileBytes, err := ioutil.ReadFile("../../samples/cert.pem")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	_, err = restyClient.R().
+		SetBody(string(certFileBytes)).
+		SetContentLength(true).
+		Post(fmt.Sprintf("%s%s", endpoint, certificateAlias))
```

Comment:
Let's rename `endpoint` to a descriptive name.

Reply:
Was reusing the same variable already defined in `resource_artifactory_certificate.go` file. Changed the variable name to `certificateEndpoint`